### PR TITLE
DOC-85 Migrate howto articles

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

File: content/rs/references/rladmin.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- Tune specific internal parameters of a database
+- Upgrade the version of a database to the most current version supported by the cluster
+
+## Using the rladmin CLI
+
+To open the rladmin CLI:
+
+1. Login to the RS host with an account that is either:
+    - a member of the **redislabs** OS group
+    - change user to root with command: `sudo su --`
```
Comment (bmansheim) [inline content/rs/references/rladmin.md]: @Paz-Yanover Remove this line?

File: content/rs/references/rlcheck.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+This utility can be useful to confirm a successful installation or to verify that the node is functioning properly.
+
+You can access rlcheck from the host command-line interface (CLI).
+The output of rlcheck shows information specific to the host that you run it on.
+
+To open the rladmin CLI:
+
+1. Login to the RS host with an account that is either:
+    - a member of the **redislabs** OS group
+    - change user to root with command: `sudo su --`
```
Comment (bmansheim) [inline content/rs/references/rlcheck.md]: @Paz-Yanover Remove this line?

File: content/rs/references/memtier-benchmark.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+--ratio=1:1 --distinct-client-seed --randomize --test-time=600
+--run-count=1 --out-file=test.out
+```
+
+Where:
+
+|  **Parameter** | **Description** |
+|------------|-----------------|
+|  Access pattern (--key-pattern) and standard deviation (--key-stddev) | Controls the RAM Hit ratio after the centralization process is complete |
+|  Number of threads\ | Controls how many connections are opened to the database, whereby the number of connections is the number of threads multiplied by the number of connections per thread (-t) and number of clients per thread (-c) |
```
Comment (bmansheim) [inline content/rs/references/memtier-benchmark.md]: @Paz-Yanover Do you know why there are 3 entries for Number of Threads?

File: content/rs/references/rladmin.md (hunk 1 above)
Comment (Paz-Yanover) [reply content/rs/references/rladmin.md]: @bmansheim - can you please the history of this change? I recall we had some requests to had the sudo. If there is no relevant history (and explanations), let's remove it

File: content/rs/references/rlcheck.md (hunk 2 above)
Comment (Paz-Yanover) [reply content/rs/references/rlcheck.md]: @bmansheim - can you please the history of this change? I recall we had some requests to had the sudo. If there is no relevant history (and explanations), let's remove it

File: content/rs/references/memtier-benchmark.md (hunk 3 above)
Comment (Paz-Yanover) [reply content/rs/references/memtier-benchmark.md]: @bmansheim - no. Please consult yaacov.hazan@redislabs.com

Comment (bmansheim) [reply content/rs/references/memtier-benchmark.md]: @yaacov.hazan@redislabs.com Can you explain?

File: content/rs/references/rlcheck.md (hunk 2 above)
Comment (bmansheim) [reply content/rs/references/rlcheck.md]: @Paz-Yanover I don't see any specific requests. I'll remove it.

File: content/rs/references/rladmin.md (hunk 1 above)
Comment (bmansheim) [reply content/rs/references/rladmin.md]: @Paz-Yanover I don't see any request for this so I will remove it.

File: content/rs/references/memtier-benchmark.md (hunk 3 above)
Comment (bmansheim) [reply content/rs/references/memtier-benchmark.md]: Rows are mislabeled.

**Pull request merged**